modules/aws_iam_policy/tf_module/main.tf

```diff
@@ -0,0 +1,13 @@
+resource "random_string" "policy_suffix" {
+  length  = 4
+  special = false
+  upper   = false
+}
+
+resource "aws_iam_policy" "policy" {
+  name   = "${var.env_name}-${var.layer_name}-${var.module_name}"
+  policy = file(var.file)
```

Comment:
we didn't need to do anything fancy for the relative file path, right? (check other examples)